### PR TITLE
MGN12 Carriage Mod

### PR DESCRIPTION
**Pull request by BlueDragonX**
This adds a modified Arkeet MGN12 X carriage for the V2.4. Adjusted dock arms are included in order to properly position the probe dock. These have been tested on my V2.4.

### CODE REVIEW

**Comment by jlas1** [comment]
Great work, let me just check the files on my laptop.

File: Usermods/bluedragonx/README.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+This mod includes an MGN12 X carriage for the Voron 2.4 that directly
+integrates the Klicky mount. This is based on [Arkeet's MGN12 carriage][1].
+
+You will need to print both left and right halfs and install the magnets and
+wiring as you would the regular Klicky mount.
+
+The Klicky probe sits higher on the carriage with this mod. The standard fixed
+docks will not work. You will need to print either the updated fixed mount or
+updated side mount depending on your desired dock position.
+
```
Comment (jlas1) [inline Usermods/bluedragonx/README.md:10]: Can you add what is the necessary length distance, it would make it easier to change other custom docks

**Review by jlas1 — COMMENTED** [review]
Can you add something like "x_carriage_" to the beginning of the probe docks mounts, to make it easier if a user downloads all the stls at the same time.

File: Usermods/bluedragonx/README.md (hunk 1 above)
Comment (BlueDragonX) [reply Usermods/bluedragonx/README.md:10]: I actually don't know what that is since I use my own macros. How would I determine that?

**Comment by BlueDragonX** [comment]
Done. I actually wanted to rename them all since this is specific to the V2.4 and I may do a similar mod for other printers.

File: Usermods/bluedragonx/README.md (hunk 1 above)
Comment (BlueDragonX) [reply Usermods/bluedragonx/README.md:39]: Done.

**Review by jlas1 — APPROVED** [review]
Great work, i'll merge them

**Pull request merged**